<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-07-02T11:00:39 IST" name="Suite" finished-at="2024-07-02T11:01:12 IST" duration-ms="33885">
    <groups>
    </groups>
    <test started-at="2024-07-02T11:00:39 IST" name="Test" finished-at="2024-07-02T11:01:12 IST" duration-ms="33885">
      <class name="com.kiwiqa.testreport.TestClass">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:com.kiwiqa.testreport.TestClass@222545dc]" started-at="2024-07-02T11:00:39 IST" name="beforeTest" finished-at="2024-07-02T11:00:44 IST" duration-ms="5211" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method signature="loginVerify()[pri:0, instance:com.kiwiqa.testreport.TestClass@222545dc]" started-at="2024-07-02T11:00:44 IST" name="loginVerify" finished-at="2024-07-02T11:01:08 IST" duration-ms="24057" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Successfully Updated] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Successfully Updated] but found []
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.kiwiqa.orangehrmpages.MyInfoPageMethod2.verifySuccessfullyUpdate(MyInfoPageMethod2.java:88)
at com.kiwiqa.testreport.TestClass.loginVerify(TestClass.java:77)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginVerify -->
        <test-method is-config="true" signature="afterMethod(org.testng.ITestResult)[pri:0, instance:com.kiwiqa.testreport.TestClass@222545dc]" started-at="2024-07-02T11:01:08 IST" name="afterMethod" finished-at="2024-07-02T11:01:11 IST" duration-ms="3565" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginVerify status=FAILURE method=TestClass.loginVerify()[pri:0, instance:com.kiwiqa.testreport.TestClass@222545dc] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterclass()[pri:0, instance:com.kiwiqa.testreport.TestClass@222545dc]" started-at="2024-07-02T11:01:11 IST" name="afterclass" finished-at="2024-07-02T11:01:12 IST" duration-ms="983" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterclass -->
      </class> <!-- com.kiwiqa.testreport.TestClass -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
